isso/views/comments.py

```diff
@@ -561,25 +561,19 @@ def delete(self, environ, request, id, key=None):
     @apiParam {string} email
         The email address of the subscriber.
     @apiParam {string} key
-        The key to authenticate the subscriber.
+        The key to authenticate the subscriber. Containing a list with the keyword unsubscribe and the email ['unsubscribe', email].
```

Comment:
Maybe mention here that the key is also base64-encoded.
The stray `['unsubscribe', email]` here is also a bit confusing.
Maybe put it on a new line with "Expected syntax" or something.

Reply:
Pushed to your branch with my suggested change. I've left the base64 encoding part out because there's also a signature attached, which would just be confusing.

Reply:
thanks :+1: